core/src/main/java/de/danoeh/antennapod/core/preferences/PlaybackPreferences.java

```diff
@@ -43,6 +43,9 @@
 	/** True if last played media was a video. */
 	public static final String PREF_CURRENT_EPISODE_IS_VIDEO = "de.danoeh.antennapod.preferences.lastIsVideo";
 
+	/** Value of PREF_QUEUE_ADD_TO_FRONT if no media is playing. */
+	public static final String PREF_QUEUE_ADD_TO_FRONT = "prefQueueAddToFront";
```

Comment:
including the app module as a dependency of the core module cause a circular dependency.